src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -533,6 +533,9 @@ void PeakDetectorCLI::loadCompoundsFile() {
 		int loadCount = DB.loadCompoundCSVFile(mavenParameters->ligandDbFilename);
```

Comment:
@saifulbkhan 
Have you tested this change in different scenarios ?
In some cases, the CLI will crash before reaching your changes, while executing the 2 lines below.
You should make sure that this piece of code doesn't crash.

Reply:
I have tested this for two cases:
1. When CSV file has only headers and no entries.
2. When CSV file is completely empty.

There is a scenario where, if the CSV file has blank spaces and newlines, our parser crashes the application. But this is an issue that is not related to this PR. We need to make some changes to our parser to catch these edge cases.